### DISCUSSION TITLE
System.Drawing.Image to ZXing.Net with NET6.0

### DISCUSSION THREAD

**Discussion by khr245** [micjahn/ZXing.Net#398] [Q&A] [answered]
How to pass System.Drawing.Image to ZXing.Net with net6.0
I'm creating a QR code reader now, but when I try to pass System.Drawing.Image to ZXing with NET6.0, I get an error.
What should I do best? Ask for improvement measures.

**Comment by micjahn** [comment, marked as answer]
Without knowing the concrete error message I think you should add the following nuget package:
https://www.nuget.org/packages/ZXing.Net.Bindings.Windows.Compatibility/
Then create your reader 
`var reader = new ZXing.Windows.Compatibility.BarcodeReader()`

**Reply by khr245** [reply]
Thank you.
